commands/request.go

```diff
@@ -28,8 +28,9 @@ type Context struct {
 	config     *config.Config
 	LoadConfig func(path string) (*config.Config, error)
 
-	node          *core.IpfsNode
-	ConstructNode func() (*core.IpfsNode, error)
+	node                     *core.IpfsNode
+	ConstructNode            func() (*core.IpfsNode, error)
+	ContextIsReadyToBeClosed chan bool
```

Comment:
Yep, I chose to use `ContextIsReadyToBeClosed` because it looked ugly and what I proposed was just a temporary solution. `InitDone`, so be it.